tests/test_quantity.py

```diff
@@ -255,14 +255,14 @@ def test_to_29():
     v = ansunits.Quantity(1.0, "BTU lb^-1 R^-1")
     to = v.to("J kg^-1 K^-1")
     assert to.value == pytest.approx(4186.8161854, DELTA)
-    assert to.units == "J kg^-1 delta_K^-1"
+    assert to.units == "J kg^-1 K^-1"
 
 
 def test_to_30():
     v = ansunits.Quantity(1.0, "BTU lb^-1 F^-1")
     to = v.to("J kg^-1 K^-1")
     assert to.value == pytest.approx(4186.8161854, DELTA)
-    assert to.units == "J kg^-1 delta_K^-1"
+    assert to.units == "J kg^-1 K^-1"
```

Comment:
Not sure about this change.

Reply:
I can try and fix it. Should all K^-1 be delta_K^-1?

Reply:
We're in a murky area, but I'm reluctant to wave through changes to existing tests. I'm hoping we can add the new feature while preserving behaviour.

Reply:
Once powers are involved with the units I believe they will be type "Composite". For this example I am a bit confused why it would be a delta_k.  After `to = v.to("J kg^-1 K^-1")`  instead of `to = v.to("J kg^-1 delta_K^-1")`

Reply:
In these particular tests I’d expect K^-1 not delta K

Reply:
It should be `"J kg^-1 K^-1"`. We should remove temporary hack introduced to get `deltaK^-1` @seanpearsonuk . As we discussed this is not permanent solution.  Our old version was based on rules given by @h-krishnan. If we want different results then we should have to define rules first.

Reply:
User should get same unit given in `to()` method after conversion. This is one of the simplest rule that we should follow.

Reply:
I was only commenting that test assertions were changing in this PR, which I did not want to see since I thought that the purpose of this PR was only to a new feature. I was unsure of the history here regarding deltaK. When was deltaK introduced at all?

Reply:
@seanpearsonuk 

`delta_K` is introduced in this PR https://github.com/ansys/pyansys-units/pull/27 as temporary hack without proper rule definition. 

I realized that we are returning different unit than given in `to()` method.

![image](https://github.com/ansys/pyansys-units/assets/106588300/a1af4f7e-1a10-41dc-b687-6c1ece0484ac)

Reply:
@seanpearsonuk 

It was originally introduced in this PR https://github.com/ansys/pyfluent/pull/1089/files.

`Do we even need it at all?` - No. We can simply remove units involving `delta_`.

Here is the summary of how temperature units have been evolved.

1. First PR was completely consistent with CFX. 
Temperature -> K
Temperature Difference -> K

2. Later on we decided to add attribute `type` to identity temperature or temperature difference only, based on rule given by Krishnan as follows.

For explicit `"C", "K", "F", "R"` unit should be treated as temperature and e.g `t = Quantity(25, "K")`

in all other combination of these unit terms it should be treated as temperature difference e.g `t = Quantity(25, "kJ kg^-1 K^-1")`

at this stage everything involving K^-1, K^2 is temp diff.

3. Then we decided to implement `delta_K and similar units to create temp. diff. based on following rule
K^-1 = delta_K (same unit with different symbol just to have clarity that it is temp. diff)

If unit contains `delta_` then type should be temp. diff.

4. After that we started returning `delta_K^-1` instead of K^-1 by temporary hack.

**We should stop at point 2 only. because Pint is also not handling `delta_K` for various reasons.**

Reply:
OK, thanks, @hpohekar

Reply:
Fluent does not have `delta_K` therefore we can not pass or get this unit from it.

Reply:
ok but lets do that in a next step